mita/openstack.py

```diff
@@ -382,12 +382,20 @@ def get_volume(self, name: str) -> StorageVolume:
 
     def create_node(self):
         """Create the instance with the provided data."""
-        self._create_vm_node()
-        self._wait_until_vm_state_running()
-        self._wait_until_ip_is_known()
-
-        if self.no_of_volumes:
-            self._create_attach_volumes()
+        try:
+            self._create_vm_node()
+            self._wait_until_vm_state_running()
+            self._wait_until_ip_is_known()
+
+            if self.no_of_volumes:
+                self._create_attach_volumes()
+        except (ResourceNotFound, NodeErrorState, GetIPError, VolumeOpError):
```

Comment:
can we also get exception messages here. those exception classes don't have anything except `pass` :)

Reply:
You are correct. There is no default exception message for these classes. However, the places where we raise the exceptions in our code have appropriate messages. In this place, I am trying not to override their messages with a generic message. The exception message may vary...